resources/js/components/GlobalSearch.vue

```diff
@@ -116,7 +127,7 @@ export default {
         },
         changeTab (item) {
             this.activeTab = item;
-            setTimeout(() => this.createObserver(), 0);
+            setTimeout(() => this.createObserver(), 0);// need time to render DOM after switching tab
```

Comment:
is it possible to use `this.$nextTick()` instead of setTimeout?

Reply:
Yes, It is